elements/pl-order-blocks/pl-order-blocks.mustache

```diff
@@ -62,7 +62,12 @@ $(function() {
         </ul>
         <ul class="list-group submission_table">
             {{#student_submission}}
-            	<li class="{{block_formatting}}" style="margin-left: {{indent}}px">{{{inner_html}}}</li>
+            	<li class="{{block_formatting}}" style="margin-left: {{indent}}px">
+                <div class="badge {{badge_type}}" style='display: table-cell; vertical-align: middle;'>
```

Comment:
The badge should probably be in a `span`, not a `div`, because its supposed to be inline. Once you do that, you should have to have the `display: table-cell` hack.

Reply:
oh wait, you might still have a problem with it showing inline in the case of `format=code`, because of the fact that `pl-code` always get translated into a `div`. If there _is_ a `pl-code` specific hack that is needed, it should probably go over here: 
https://github.com/PrairieLearn/PrairieLearn/blob/00fe7665ac9c16ca94d650c5b48f2805af289447/elements/pl-order-blocks/pl-order-blocks.css#L19-L22
because that's where we already have one such hack.